src/gui/owncloudgui.cpp

```diff
@@ -713,5 +712,17 @@ void ownCloudGui::slotRemoveDestroyedShareDialogs()
     }
 }
 
+void ownCloudGui::updateDialogVisibility(QDialog *dialog, bool visible)
+{
+    if (dialog) {
```

Comment:
Honestly this shouldn't receive a nullptr, so I'd make that the responsibility of the caller and use a Q_ASSERT here.